src/api/utils/errors.ts

```diff
@@ -3,9 +3,9 @@ import type { Response } from '@vercel/fetch'
 // Used for GraphQL errors
 export class BigcommerceGraphQLError extends Error {
   constructor(msg: string) {
-    super(msg)
-    // Necessary to compile to ES5: https://github.com/microsoft/TypeScript/issues/22585
-    Object.setPrototypeOf(this, BigcommerceGraphQLError.prototype);
+    super()
+
+    this.message = msg
```

Comment:
🍹  can this be moved to the constructor? `constructor(private readonly msg: string)` should auto initialize it